src/App.tsx

```diff
@@ -1,16 +1,14 @@
 import { useState, useEffect } from 'react'
-import { BrowserRouter } from 'react-router-dom'
-import Footer from './Components/Footer/Footer'
+import { HashRouter } from 'react-router-dom'
 import Loading from './Components/Loading/Loading'
-import Navbar from './Components/Navbar/Navbar'
 import Navigation from './pages/Navigation'
 
 const Page = () => {
   return (
     <>
-      <BrowserRouter>
+      <HashRouter>
         <Navigation />
-      </BrowserRouter>
+      </HashRouter>
```

Comment:
something is wrong with this implementation as it breaks the routes on clicking links (try clicking Schedule and then Login, observe the address bar)